src/frontend/screens/Settings/MapManagement/BackgroundMaps.tsx

```diff
@@ -0,0 +1,336 @@
+        <Text style={styles.aboutText}>{t(m.about)}</Text>
+        <View style={styles.descriptionContainer}>
+          <Text>{t(m.description1)}</Text>
+          <Text>{t(m.description2)}</Text>
+        </View>
+
+        <CustomMapInfoSection
+          onChooseFile={() => {
+            selectFileMutation.mutate(
+              {
+                extensionFilters: ['smp'],
+              },
+              {
+                onSuccess: asset => {
+                  if (!asset) return;
+
+                  importCustomMapMutation.mutate(
+                    {
+                      uri: asset.uri,
+                    },
+                    {
+                      onError: () => {
+                        FileSystem.deleteAsync(asset.uri, {
+                          idempotent: true,
+                        }).catch(noop);
+                      },
+                      onSuccess: () => {
+                        mapAddedBottomSheet.openSheet();
+                      },
+                    },
+                  );
+                },
+              },
+            );
+          }}
+          onRemoveMap={() => {
+            removeMapBottomSheet.openSheet();
+          }}
+        />
+
+        {customMapInfoQuery.status === 'error' && (
+          <>
+            <Text style={styles.infoLoadErrorText}>
+              {t(m.customMapInfoLoadError)}
+            </Text>
+            <Button
+              fullWidth
+              variant="outlined"
+              onPress={() => {
+                removeCustomMapMutation.mutate();
+              }}>
+              <Text style={styles.removeMapFileButtonText}>
+                {t(m.removeMapFile)}
+              </Text>
+            </Button>
+          </>
+        )}
+      </ScrollView>
+
+      <BottomSheetModal
+        ref={removeMapBottomSheet.sheetRef}
+        isOpen={removeMapBottomSheet.isOpen}>
+        <BottomSheetModalContent
+          icon={<ErrorSvg />}
+          title={t(m.deleteCustomMapTitle)}
+          description={
+            t(m.deleteCustomMapDescription) + '\n\n' + t(m.cannotBeUndone)
+          }
+          buttonConfigs={[
+            {
+              dangerous: true,
+              variation: 'filled',
+              text: t(m.deleteMapButtonText),
+              icon: <MaterialIcon size={30} name="delete" color={WHITE} />,
+              onPress: () => {
+                removeCustomMapMutation.mutate();
+                removeMapBottomSheet.closeSheet();
+              },
+            },
+            {
+              variation: 'outlined',
+              text: t(m.close),
+              onPress: () => {
+                removeMapBottomSheet.closeSheet();
+              },
+            },
+          ]}
+        />
+      </BottomSheetModal>
+
+      <BottomSheetModal
+        fullScreen
+        ref={mapAddedBottomSheet.sheetRef}
+        isOpen={mapAddedBottomSheet.isOpen}>
+        <BottomSheetModalContent
+          icon={<GreenCheckSvg />}
```

Comment:
yeah this ideally is fixed in the `BottomSheetModalContent` component but addressed here as a one-off for now (since we probably do the same in other places). See e65020cab5368512d268a2fe20755ba79e8b5aa1

<img width="200" alt="image" src="https://github.com/user-attachments/assets/198656b8-3380-46bc-a1ca-20fead803189">